src/routes/setting/setting.js

```diff
@@ -132,17 +147,36 @@ const form = ({
   const settings = Object.keys(settingsGrouped).sort((a, b) => {
     const categoryA = getCategoryWeight(a)
     const categoryB = getCategoryWeight(b)
-    if (categoryA < categoryB) {
-      return -1
-    }
-    if (categoryA > categoryB) {
-      return 1
-    }
-    return 0
+    return categoryA - categoryB
   }).map(item => (
     <div key={item}>
       <div className={classnames(styles.fieldset, { [styles.dangerZone]: item === 'danger Zone' })}>
         <span className={styles.fieldsetLabel}>{item}</span>
+        {item === 'danger Zone' && (
+          <Alert
+            style={{ marginBottom: '1rem', marginTop: '1rem' }}
+            message={
+            <div className={styles.description}>
+              <span>
```

Comment:
@jillian-maroket , could you kindly help review or suggest these wordings?


<img width="1488" alt="Screenshot 2024-05-27 at 5 55 05 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/06e01615-2ed6-48fe-8047-54a25c721b32">